cmake/projects/zookeeper/hunter.cmake

```diff
@@ -0,0 +1,29 @@
+# cmake/projects/Example/hunter.cmake
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    zookeeper
+    VERSION
+    "3.4.9-p1"
+    URL
+    "https://github.com/Artomatix/zookeeper/archive/3.4.9-hunter-5.tar.gz"
+    SHA1
+    f98aba57b183de9e00a58536c7a47c612e3d51fc
+)
+
+# Pick a download scheme
+hunter_pick_scheme(DEFAULT url_sha1_cmake) # use scheme for cmake projects
+
+# Download package.
+# Two versions of library will be build by default:
+#     * libexample_A.a
+#     * libexample_Ad.a
```

Comment:
Is this package relocatable? https://docs.hunter.sh/en/latest/creating-new/cmake.html#important